modules/ROOT/pages/_partials/nav-dw.adoc

```diff
@@ -133,34 +156,62 @@
 **** xref:dw-objects-functions-someentry.adoc[someEntry]
 **** xref:dw-objects-functions-takewhile.adoc[takeWhile]
 **** xref:dw-objects-functions-valueset.adoc[valueSet]
+*** xref:dw-periods.adoc[Periods (dw::core::Periods)]
+**** xref:dw-periods-functions-between.adoc[between]
```

Comment:
ACTION: copyedited BUT there are implementation details in the description. Not clear if all of that info is really necessary.

TODO: talk to MDA about implementation details

PR: TODO